utils/programs/asa_clear.teal

```diff
@@ -0,0 +1,9 @@
+byte base64 Ymw=
```

Comment:
An overview similar to utils/programs/globcheck.teal  with be super helpful here.